packages/wallet-service/package.json

```diff
@@ -72,5 +68,12 @@
     "typescript-eslint": "0.0.1-alpha.0",
     "webpack": "^5.88.2",
     "webpack-node-externals": "^3.0.0"
+  },
+  "peerDependencies": {
+    "@aws-sdk/client-apigatewaymanagementapi": "3.540.0",
+    "@aws-sdk/client-lambda": "3.540.0",
+    "@aws-sdk/client-sqs": "3.540.0",
+    "@hathor/wallet-lib": "^0.39.0",
+    "winston": "^3.13.0"
```

Comment:
Any reason to upgrade this package?

Reply:
Since we were using caret ranges in the version (`^3.3.0`), we were already installing the latest non-major version (< `4.0.0`) after a `yarn install`

![image](https://github.com/HathorNetwork/hathor-wallet-service/assets/3586068/54c445c3-b189-4d75-b225-68c71e58097e)

![image](https://github.com/HathorNetwork/hathor-wallet-service/assets/3586068/0cd7c3ea-163f-4bdb-85af-a5b2fbcff9b0)

So this will not cause the package to get updated, it's just setting the version we're actually using